From 62b8a6b57009c6723c9ca50bee6219dd6dd4d633 Mon Sep 17 00:00:00 2001
From: hmz007 <hmz007@gmail.com>
Date: Fri, 25 Dec 2020 15:38:08 +0800
Subject: [PATCH 67/73] arm64: configs: refresh defconfig for nanopi4/r2

After savedefconfig, some options for CROS_EC are removed
(actually they are not needed).

Signed-off-by: hmz007 <hmz007@gmail.com>
---
 arch/arm64/configs/nanopi-r2_linux_defconfig | 10 ++--------
 arch/arm64/configs/nanopi4_linux_defconfig   | 10 ++--------
 2 files changed, 4 insertions(+), 16 deletions(-)

diff --git a/arch/arm64/configs/nanopi-r2_linux_defconfig b/arch/arm64/configs/nanopi-r2_linux_defconfig
index fd1c6599dd2b..979f25290833 100644
--- a/arch/arm64/configs/nanopi-r2_linux_defconfig
+++ b/arch/arm64/configs/nanopi-r2_linux_defconfig
@@ -64,7 +64,6 @@ CONFIG_ARCH_ROCKCHIP=y
 # CONFIG_SOCIONEXT_SYNQUACER_PREITS is not set
 CONFIG_SCHED_MC=y
 CONFIG_NR_CPUS=8
-CONFIG_SECCOMP=y
 CONFIG_COMPAT=y
 CONFIG_ARMV8_DEPRECATED=y
 CONFIG_SWP_EMULATION=y
@@ -112,7 +111,6 @@ CONFIG_CLEANCACHE=y
 CONFIG_FRONTSWAP=y
 CONFIG_CMA=y
 CONFIG_ZSMALLOC=y
-CONFIG_ZSMALLOC_PGTABLE_MAPPING=y
 CONFIG_NET=y
 CONFIG_PACKET=y
 CONFIG_PACKET_DIAG=m
@@ -355,6 +353,7 @@ CONFIG_IP_NF_ARP_MANGLE=m
 CONFIG_NFT_DUP_IPV6=m
 CONFIG_NFT_FIB_IPV6=m
 CONFIG_NF_FLOW_TABLE_IPV6=m
+CONFIG_IP6_NF_IPTABLES=m
 CONFIG_IP6_NF_MATCH_AH=m
 CONFIG_IP6_NF_MATCH_EUI64=m
 CONFIG_IP6_NF_MATCH_FRAG=m
@@ -904,7 +903,6 @@ CONFIG_KEYBOARD_ADC=y
 CONFIG_KEYBOARD_GPIO=y
 CONFIG_KEYBOARD_GPIO_POLLED=y
 CONFIG_KEYBOARD_MATRIX=m
-CONFIG_KEYBOARD_CROS_EC=m
 # CONFIG_MOUSE_PS2 is not set
 CONFIG_MOUSE_CYAPA=m
 CONFIG_MOUSE_ELAN_I2C=m
@@ -964,7 +962,6 @@ CONFIG_I2C_MUX=y
 CONFIG_I2C_MUX_PCA954x=m
 CONFIG_I2C_GPIO=m
 CONFIG_I2C_RK3X=y
-CONFIG_I2C_CROS_EC_TUNNEL=m
 CONFIG_SPI=y
 CONFIG_SPI_BITBANG=y
 CONFIG_SPI_ROCKCHIP=y
@@ -992,8 +989,6 @@ CONFIG_W1_SLAVE_DS2780=m
 CONFIG_W1_SLAVE_DS2781=m
 CONFIG_W1_SLAVE_DS28E04=m
 CONFIG_W1_SLAVE_DS28E17=m
-CONFIG_POWER_AVS=y
-CONFIG_ROCKCHIP_IODOMAIN=y
 CONFIG_POWER_RESET_GPIO=y
 CONFIG_POWER_RESET_GPIO_RESTART=y
 CONFIG_SYSCON_REBOOT_MODE=y
@@ -1432,11 +1427,10 @@ CONFIG_FB_TFT_TLS8204=m
 CONFIG_FB_TFT_UC1701=m
 CONFIG_FB_TFT_UPD161704=m
 CONFIG_FB_TFT_WATTEROTT=m
-CONFIG_MFD_CROS_EC=m
-# CONFIG_CROS_EC_CHARDEV is not set
 CONFIG_COMMON_CLK_RK808=y
 CONFIG_MAILBOX=y
 CONFIG_ROCKCHIP_IOMMU=y
+CONFIG_ROCKCHIP_IODOMAIN=y
 CONFIG_ROCKCHIP_PM_DOMAINS=y
 CONFIG_VENDOR_FRIENDLYELEC=y
 CONFIG_DEVFREQ_GOV_PERFORMANCE=y
diff --git a/arch/arm64/configs/nanopi4_linux_defconfig b/arch/arm64/configs/nanopi4_linux_defconfig
index fd1c6599dd2b..979f25290833 100644
--- a/arch/arm64/configs/nanopi4_linux_defconfig
+++ b/arch/arm64/configs/nanopi4_linux_defconfig
@@ -64,7 +64,6 @@ CONFIG_ARCH_ROCKCHIP=y
 # CONFIG_SOCIONEXT_SYNQUACER_PREITS is not set
 CONFIG_SCHED_MC=y
 CONFIG_NR_CPUS=8
-CONFIG_SECCOMP=y
 CONFIG_COMPAT=y
 CONFIG_ARMV8_DEPRECATED=y
 CONFIG_SWP_EMULATION=y
@@ -112,7 +111,6 @@ CONFIG_CLEANCACHE=y
 CONFIG_FRONTSWAP=y
 CONFIG_CMA=y
 CONFIG_ZSMALLOC=y
-CONFIG_ZSMALLOC_PGTABLE_MAPPING=y
 CONFIG_NET=y
 CONFIG_PACKET=y
 CONFIG_PACKET_DIAG=m
@@ -355,6 +353,7 @@ CONFIG_IP_NF_ARP_MANGLE=m
 CONFIG_NFT_DUP_IPV6=m
 CONFIG_NFT_FIB_IPV6=m
 CONFIG_NF_FLOW_TABLE_IPV6=m
+CONFIG_IP6_NF_IPTABLES=m
 CONFIG_IP6_NF_MATCH_AH=m
 CONFIG_IP6_NF_MATCH_EUI64=m
 CONFIG_IP6_NF_MATCH_FRAG=m
@@ -904,7 +903,6 @@ CONFIG_KEYBOARD_ADC=y
 CONFIG_KEYBOARD_GPIO=y
 CONFIG_KEYBOARD_GPIO_POLLED=y
 CONFIG_KEYBOARD_MATRIX=m
-CONFIG_KEYBOARD_CROS_EC=m
 # CONFIG_MOUSE_PS2 is not set
 CONFIG_MOUSE_CYAPA=m
 CONFIG_MOUSE_ELAN_I2C=m
@@ -964,7 +962,6 @@ CONFIG_I2C_MUX=y
 CONFIG_I2C_MUX_PCA954x=m
 CONFIG_I2C_GPIO=m
 CONFIG_I2C_RK3X=y
-CONFIG_I2C_CROS_EC_TUNNEL=m
 CONFIG_SPI=y
 CONFIG_SPI_BITBANG=y
 CONFIG_SPI_ROCKCHIP=y
@@ -992,8 +989,6 @@ CONFIG_W1_SLAVE_DS2780=m
 CONFIG_W1_SLAVE_DS2781=m
 CONFIG_W1_SLAVE_DS28E04=m
 CONFIG_W1_SLAVE_DS28E17=m
-CONFIG_POWER_AVS=y
-CONFIG_ROCKCHIP_IODOMAIN=y
 CONFIG_POWER_RESET_GPIO=y
 CONFIG_POWER_RESET_GPIO_RESTART=y
 CONFIG_SYSCON_REBOOT_MODE=y
@@ -1432,11 +1427,10 @@ CONFIG_FB_TFT_TLS8204=m
 CONFIG_FB_TFT_UC1701=m
 CONFIG_FB_TFT_UPD161704=m
 CONFIG_FB_TFT_WATTEROTT=m
-CONFIG_MFD_CROS_EC=m
-# CONFIG_CROS_EC_CHARDEV is not set
 CONFIG_COMMON_CLK_RK808=y
 CONFIG_MAILBOX=y
 CONFIG_ROCKCHIP_IOMMU=y
+CONFIG_ROCKCHIP_IODOMAIN=y
 CONFIG_ROCKCHIP_PM_DOMAINS=y
 CONFIG_VENDOR_FRIENDLYELEC=y
 CONFIG_DEVFREQ_GOV_PERFORMANCE=y
-- 
2.31.1

